package.json

```diff
@@ -57,6 +59,10 @@
     "@google-cloud/projectify": "^4.0.0",
     "@google-cloud/promisify": "^4.0.0",
     "@grpc/proto-loader": "^0.7.0",
+    "@opentelemetry/api": "^1.9.0",
+    "@opentelemetry/context-async-hooks": "^1.25.1",
+    "@opentelemetry/sdk-trace-node": "^1.25.1",
```

Comment:
This dependency is not used for the actual code and only required for tests. Is there a way we can only add this dependency as a test dependency ?

Reply:
Done, and moved to "devDependencies", thank you!